Code/Engine/GameEngine/GameApplication/Implementation/GameApplicationInit.cpp

```diff
@@ -260,13 +260,15 @@ void ezGameApplication::Init_LoadRequiredPlugins()
 
 #  ifdef BUILDSYSTEM_ENABLE_RENDERDOC_SUPPORT
   ezPlugin::LoadPlugin("ezRenderDocPlugin");
-#  endif
+#  endif 
 
   // on sandboxed platforms, we can only load data through fileserve, so enforce use of this plugin
 #  if EZ_DISABLED(EZ_SUPPORTS_UNRESTRICTED_FILE_ACCESS)
   ezPlugin::LoadPlugin("ezFileservePlugin"); // don't care if it fails to load
 #  endif
 
+  ezPlugin::LoadPlugin("ezTypeScriptPlugin");
```

Comment:
This should not be here

Reply:
I thought so too but I can't get it to load otherwise in UWP. It is marked in the project manifest but won't be loaded without it. What would be the general code to load this? Could it be that the dynamic loading is somehow broken in UWP?

Reply:
The project manifest would do exactly the same thing. This doesn't add a link dependency or anything, so if this works, but the project manifest doesn't work, then you should debug through this. Ie. remove this call, put a breakpoint in LoadPlugin, and check what happens when you go through the manifest. Maybe for some reason the call doesn't happen then.

Either way, this plugin is supposed to be optional, you should not add this line.

Reply:
Found the problem: The typescript plugin was not selected to be loaded for the project but the editor loads it regardless so it worked in the editor.